lib/digest.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import jsonld from 'jsonld';
+import {encode as base58Encode} from '@digitalcredentials/base58-universal';
+import {sha256digest} from './sha256digest.js';
+
+const URDCA_2015_ALG = 'urdca2015';
+// Multicodec table entry code 0xb403, converted to varint
+const URDCA2015_HEADER = new Uint8Array([131, 232, 2]);
+
+/**
+ * Performs a combined canonicalize + multihash operation.
+ *
+ * @param {object} options - Options hashmap.
+ * @param {object} options.input - JSON-LD object.
+ * @param {string} [options.canonAlgorithm] - Canonicalization algorithm.
+ * @param {Function} options.documentLoader - JSON-LD document loader.
+ *
+ * @returns {Promise<string>} Returns the canonhash digest, base58-encoded.
+ */
+export async function multibaseDigest({
+  input, canonAlgorithm = URDCA_2015_ALG, documentLoader
+} = {}) {
+  let normalized;
+
+  switch(canonAlgorithm) {
+    case URDCA_2015_ALG:
+      normalized = await jsonld.canonize(input, {documentLoader});
```

Comment:
Probably needs these two options:

https://github.com/digitalbazaar/jsonld-signatures/blob/02d0f63d7aa75a619233482ce9d0d4098057a5b2/lib/suites/LinkedDataSignature.js#L194-L196

Reply:
@dlongley - I was thinking about that, initially. Are you sure we'd want those? We're specifically defining our minimal `multibaseDigest` as "canonize to N-Quads, using URDNA2015". I'd vote for leaving out the options until we need them.

Reply:
@dmitrizagidulin, I don't mean allow someone to customize those options, I mean you have to set them. I don't think the default output is n-quads.